src/components/RouteWrapper.tsx

```diff
@@ -351,10 +351,10 @@ export const ROUTES: RouteProperties[] = [
         category: "Network Scanning and Enumeration",
     },
     {
-        name: "Eyewitness",
+        name: "EyeWitness",
         path: "/tools/eyewitness",
         element: <Eyewitness />,
-        description: "Eyewitness takes screenshot of important websites.",
+        description: "EyeWitness scans listed URLs and provides a HTML report about them, including screenshots",
```

Comment:
Super minor change. Please add a full stop at the end. Thank you!

- [ ] Add a full stop at the end.

Reply:
Thank you, I added the full stop.